app/src/main/java/com/example/groupsync/ui/gallery/ImagesFragment.kt

```diff
@@ -49,7 +54,8 @@ class ImagesFragment : Fragment() {
                         imageAdapter = ImageAdapter(requireContext(), uploads)
                         binding.recyclerView.adapter = imageAdapter
                     }
-                    binding.progressCircle.setVisibility(View.INVISIBLE);
+                    // TODO: This causes a null pointer exception...
+//                    binding.progressCircle.setVisibility(View.INVISIBLE);
```

Comment:
Wouldn't this fix the issue?
```suggestion
binding.progressCircle.setVisibility(View.INVISIBLE!!);
```